components/channel_invite_modal/channel_invite_modal.test.jsx

```diff
@@ -38,6 +38,7 @@ describe('components/channel_invite_modal', () => {
     const baseProps = {
         channel,
         profilesNotInCurrentChannel: [],
+        userStatuses: {},
```

Comment:
Would you be able to add some tests that test that:
1. Status icons are rendered correctly. This could be a snapshot test like the other ones in this file
2. Statuses are loaded when necessary.